frontend/test/metabase/components/PasswordReveal.unit.spec.js

```diff
@@ -1,25 +1,24 @@
-import { click } from "__support__/enzyme";
-
 import React from "react";
+import "@testing-library/jest-dom/extend-expect";
+import { render, fireEvent, cleanup } from "@testing-library/react";
 import PasswordReveal from "metabase/components/PasswordReveal";
-import CopyButton from "metabase/components/CopyButton";
 
-import { shallow } from "enzyme";
+// This shouldn't be needed from version 9.0.0 of react-testing-library (TODO: remove after update)
+afterEach(cleanup);
 
 describe("password reveal", () => {
-  let wrapper;
-
-  beforeEach(() => {
-    wrapper = shallow(<PasswordReveal />);
-  });
-
   it("should toggle the visibility state when hide / show are clicked", () => {
-    expect(wrapper.state().visible).toEqual(false);
-    click(wrapper.find("a"));
-    expect(wrapper.state().visible).toEqual(true);
+    const { getByTestId } = render(<PasswordReveal />);
+    const toggle = getByTestId("visibility-toggle");
+
+    expect(toggle.textContent).toEqual("Show");
+    fireEvent.click(toggle);
+    expect(toggle.textContent).toEqual("Hide");
   });
 
   it("should render a copy button", () => {
-    expect(wrapper.find(CopyButton).length).toEqual(1);
+    const { getByTestId } = render(<PasswordReveal />);
+    // explicit assertion, but it requires "@testing-library/jest-dom/extend-expect";
+    expect(getByTestId("copy-button")).toBeInTheDocument();
```

Comment:
What does the explicit assertion add over having only `getByTestId("copy-button")`? Is it for readability or something else?